data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -974,11 +974,11 @@
 
                 [sound]
                     name=heal.wav
-                [/sound]
+                [/sound]’
```

Comment:
This is why the checks are failing. You can see line 977 mentioned quite clearly in the reports:
https://github.com/wesnoth/wesnoth/actions/runs/5131783568/jobs/9232280002
https://github.com/wesnoth/wesnoth/actions/runs/5131783568/jobs/9232280117